NAMESPACE

```diff
@@ -82,10 +83,20 @@ export(use_tidy_support)
 export(use_tidy_thanks)
 export(use_tidy_versions)
 export(use_travis)
+export(use_travis_deploy)
 export(use_usethis)
 export(use_version)
 export(use_vignette)
 export(write_over)
 export(write_union)
 import(fs)
 importFrom(glue,glue)
+importFrom(openssl,rsa_keygen)
```

Comment:
The `importFrom()` calls here contradict the coding style of the rest of the package.

Reply:
Yes, was still an importFrom leftover. Cleared now.